### DISCUSSION TITLE
Praise for surveydown 🎉 🙏

### DISCUSSION THREAD

**Discussion by jhelvy** [surveydown-dev/surveydown#144] [Announcements]
Showing evidence of the impact that open source software such as {surveydown} has on the world is hard to accomplish because the typical metrics that matter (e.g., publications, grants, citations, etc.) don't really apply. The consequence is fewer incentives and resources to develop packages like {surveydown} 😞

The good news is you can help! If you have found {surveydown} useful in any regard, please leave feedback here for other users, funders, and employers to view. This helps the package authors show how {surveydown} is being used by academic and non-academic users to increase their productivity and work quality.

So please contribute some praise 😁! Tell us how cool this package is and how you use it in your work!

And if you write a paper that used {surveydown}, please cite the package in your publications (run `citation("surveydown")` in R)

**Comment by chris-gaskell** [comment]
Amazing package that has opened a world of options for my future projects. Authors are highly responsive and attentive to maintaining and developing the package further.

**Comment by bastianolea** [comment]
This is really simple to set up and get it working. Also, the demos are very useful to get straight into a working example. Documentation is good also, I got my Supabase DB running in minutes! Thank you so much!

**Comment by willking98** [comment]
I have now completed two large (n>1000) choice based conjoint surveys with `Surveydown`. A massive congratulations to the `Surveydown` development team for building such a useful package and being on hand to answer queries and introduce new functionality during the development process. I will of course be citing and acknowledging `Surveydown` during the dissemination of my work. Thank you @jhelvy, @pingfan-hu and @Buneabt for your work on this!

**Comment by jhelvy** [comment]
> I will of course be citing and acknowledging `Surveydown`

Thank you! 🙏 Btw, we're working on a paper that will make it easier to cite. If all goes well, hoping for a publication sometime in 2025.

**Reply by Emberwhirl** [reply]
> Thank you! 🙏 Btw, we're working on a paper that will make it easier to cite. If all goes well, hoping for a publication sometime in 2025.

Thank you for developing such a nice package. Interesting to know if there is any update on publication, either in journals or preprint servers, so that I can have the privilege to cite it in a recent manuscript.

**Reply by jhelvy** [reply]
Thanks! No paper to cite yet (we're still writing it), but for now you can cite the package. You can get the citation information for any package with the `citation()` function in R, so in this case:

```r
citation("surveydown")
```

will give:

```
To cite surveydown in publications use:

  Helveston J, Hu P, Bunea B (2024). _surveydown: Markdown-Based
  Surveys Using Quarto And Shiny_. R package,
  <https://surveydown.org/>.

A BibTeX entry for LaTeX users is

  @Manual{,
    title = {surveydown: Markdown-Based Surveys Using Quarto And Shiny},
    author = {John Paul Helveston and Pingfan Hu and Bogdan Bunea},
    year = {2024},
    note = {R package},
    url = {https://surveydown.org/},
  }
```

**Reply by jtkulas** [reply]
Thanks for all your attention and perseverence on this!! 

Very much appreciated

I've used it for class surveys (how's the semester going, etc) and am just starting to port over other projects... I'm currently piloting a project that is retaining response latency as a (possible) indicator, so the default time(s) recorded are appreciated when they have historically been primarily restricted to lab-based software/hardware (reliability of these times with potential server/browser lag is likely to happen but for my purposes a little error is tolerable).

**Comment by saudiwin** [comment]
This isn't actually a feature request. Just wanted to say I'm super impressed with this package, its development, and the impact it will have on the wider research community. Am preparing now for its first deployment to survey research in Africa and the Middle East. Will post when I have findings!!

**Comment by annececile22** [comment]
Just a short message to sincerely thank you for all the work put in this package. The documentation is very clear, I managed to create a simple questionnaire in just an hour or so. Excited about using it for my next research project.

**Comment by kpivert** [comment]
[John](@jhelvy) and [Pingfan](@pingfan-hu),
Again, great to meet you IRL in ATL. Excited that I finally have a use case for {surveydown}, and it works beautifully. The documentation is really well done. Just FYI, I ran into a (totally user-based) snag because I didn't have the .env file in the same directory as the `app.R` and `survey.qmd.` Everything was connected, it just didn't record the responses. Completely resolved after moving the file. Of course, will cite you in the end product. Thank you again for all your hard work on this. 
Kurtis

**Reply by jhelvy** [reply]
Thanks! We now have a PLOS-One publication you can cite :)
You can use `citation('surveydown')` to get the reference

**Comment by Hergie** [comment]
With no experience in survey programming or Quarto, I managed to create a survey with randomized treatments and various question types in a few hours. This is a fantastic resource for applied researchers with a preference for R-based approaches. Thank you!

**Comment by virginiascarlett** [comment]
I found out about surveydown through my colleague David Lippert at the GW OSPO. My university has a Qualtrics license, but I've started using surveydown instead so that my entire workflow is transparent and reproducible. I'm telling all my colleagues about it, and maybe one day we can get my university to stop paying for a closed-source, proprietary survey tool!

I've never created a shiny app or a relational database before, so I'm learning a lot of new skills, and surveydown made it super easy to get started. The documentation is of above-average quality, the maintainer is very responsive, and the tool is free and easy-to-use. I look forward to using it in a scholarly study.

**Reply by jhelvy** [reply]
Woohoo, thanks!
